man1/aur-fetch-snapshot.1

```diff
@@ -0,0 +1,32 @@
+.TH AUR-FETCH 1 2018-02-14 AURUTILS
+.SH NAME
+aur-fetch-snapshot \- fetches a tar archive from a given URL.
+
+.SH SYNOPSIS
+.SY aur
+.B fetch
+.B snapshot
```

Comment:
`fetch-snapshot`

Maybe at some point subcommands should be nested, right now you still need the dash.